src/gridstack.scss

```diff
@@ -88,20 +88,8 @@ $animation_speed: .3s !default;
       }
     }
 
-    @for $i from 0 through $gridstack-columns {
-      &[gs-w='#{$i}'] { width: math.div(100%, $gridstack-columns) * $i; }
-      &[gs-x='#{$i}'] { left: math.div(100%, $gridstack-columns) * $i; }
-      &[gs-min-w='#{$i}'] { min-width: math.div(100%, $gridstack-columns) * $i; }
-      &[gs-max-w='#{$i}'] { max-width: math.div(100%, $gridstack-columns) * $i; }
-    }
-  }
-
-  &.grid-stack-1>.grid-stack-item {
-    min-width: 100%;
-    &[gs-w='1'] { width: 100%; }
-    &[gs-x='1'] { left: 100%; }
-    &[gs-min-w='1'] { min-width: 100%; }
```

Comment:
where are you doing min/max then ?

Reply:
Yes need to handle min, max. I tried this idea and thought you would implement this in the lib.

Reply:
toughts wrong... I have a billions things to do, this isn't on top of my list. if you want this it needs to be a complete solution and test with >1k items before vs after.